audioipc/src/lib.rs

```diff
@@ -76,7 +80,8 @@ impl serde::Serialize for PlatformHandle {
     where
         S: serde::Serializer,
     {
-        serializer.serialize_i64(self.0 as i64)
+        let h = self.0.borrow().unwrap_or(INVALID_HANDLE_VALUE);
+        serializer.serialize_i64(h as i64)
```

Comment:
What happens if `INVALID_HANDLE_VALUE` is passed from one side to another? Would this _handle_ be checked before it's converted into a _file_?

Reply:
I'll make this a panic instead, it's a bug if this happens.